src/service/viewer-impl.js

```diff
@@ -263,6 +264,12 @@ export class Viewer {
         || this.isWebviewEmbedded_
         || !ampdoc.isSingleDoc());
 
+    /**
+     * Whether the AMP document is embedded in a Chrome Custom Tab.
+     * @private @const {boolean}
+     */
+    this.isCustomTab_ = this.win.location.search.indexOf('amp_agsa=1') != -1;
```

Comment:
This can also match with any prefix and suffix e.g. `?garbage_amp_agsa=12345`. Probably want to use `parseQueryString()` here.